jax/_src/lru_cache.py

```diff
@@ -41,6 +42,8 @@ class LRUCache(CacheInterface):
   is disabled, and the LRU cache functions as a normal cache
   without any size limitations.
   """
+  _cache_suffix = "-cache.bin"
```

Comment:
I think it's fine to make these module-level constants, but feel free to keep as is if you prefer.

Reply:
Done